tests/test_ilcs.py

```diff
@@ -0,0 +1,8 @@
+import numpy
+
+
+def test_parse_1(ilcs):
+    numpy.testing.assert_almost_equal(
+        ilcs.comparator('125 55/21', '126 55/21'),
+        numpy.array([1, 0, 1, 2.1666667, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0])
```

Comment:
The overall size of this distance vector looks good to me, but you can already tell that the scalar distance representing the chapter (index 4) is off. We probably want something closer to what we do for `PriceType`, except extended so that it knows that e.g. `a` is closer to `b` than `c` (since subsections are sometimes numbered by letters).

Reply:
or an exact comparator.